src/aap_eda/core/models/activation.py

```diff
@@ -112,3 +115,8 @@ class Meta:
         choices=RulebookProcessLogLevel.choices(),
         default=get_default_log_level,
     )
+    k8s_service_name = models.TextField(
+        null=False,
+        default=get_default_k8s_service_name,
```

Comment:
I think this field can not have a default value from system settings because it must be different per activation. What if we use the name of the activation which is already unique?  In the migration we can update the existing records and ensure the new ones are created with that value.

Reply:
Why must it be different per activation? the purpose here is to allow users to choose their services. We don't need migration for the old records.

Reply:
We are creating a service per activation selecting a job_name (which is the current rulebook process) if you use the same name you would overwrite the existing one, so you can have only one service for just one activation.

Reply:
@Alex-Izquierdo I didn't get your point. Here we want to allow the user to select their pre-defined services.  Every activation can use it. And the `job_name` is changed to separate from the service name.